bin/magic-script.js

```diff
@@ -76,19 +79,20 @@ require('yargs') // eslint-disable-line
   }, argv => require('../commands/build')(argv))
   .command('remove', 'Remove project from device', argv => require('../commands/remove')(argv.argv))
   .command('run [target]', 'Compile and run project', yargs => {
+    yargs.positional('target', {
+      describe: 'target(s) to build for',
+      default: 'lumin'
+    });
     yargs.option('debug', {
       alias: 'd',
+      description: 'run in debug mode',
```

Comment:
When you type `magic-script run --help` it will show what is the default value (at least it shows on my machine). Do you want to change description anyway?